help/datagrok/solutions/domains/bio/oligo-toolkit.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Oligo Toolkit"
+sidebar_position: 1
+---
+
+The OligoToolkit is a collection of tools helping you to work with oligonucleotide sequences 
+– primarily for synthesizing DNA/RNA oligonucleotides with various chemical modifications.
+An example use-case is producing double-stranded RNAs for the RNA-silencing.
+
+To open the app, on the **Sidebar**, select **Functions** > **Apps** > **Oligo Toolkit**.
+
+![Run OligoToolkit](img/Oligotoolkit-run.gif)
+
+The OligoToolkit 
+application contains several independent modules, 
+allowing you to do the following:
+
+* Individual and batch modification of the FASTA-encoded DNA/RNA sequences according to the pattern. 
+* Conversion of chemically modified sequences between several open and commercial formats: FASTA, HELM, MerMade, BioSpring.
+* Producing chemical structures for the natural and modified oligonucleotides.
+
+## Pattern
+
+The Pattern tool applies modification patterns to a sequence of set of sequences.
+To apply modification, do the following:
+1. Create the pattern using the **Pattern** input sections.
+   * In the **Pattern** input section, choose the sequence basis (DNA/RNA) and pattern length
+     for sense and anti-sense strands.
+   * Alternatively, load the pattern from the list of saved patterns.  
+     Loading patterns created by other users are also available.
+2. Paste the nucleotide sequences (sense and/or anti-sense) in the text boxes under the pattern image, 
+   and obtain modified sequences in the Axolabs format.
+3. When you edit the pattern, OligoToolkit automatically updates the resulting sequences.
+
+![Oligotoolkit pattern](img/Oligotoolkit-pattern-edit.gif)
+
+### Batch conversion
+
+To apply one pattern to a list of nucleotides, use the **Batch conversion** mode.
+
+1. Upload/choose a dataframe containing unmodified sequences using the **Convert** input section.
+   The dataframe should contain at least three columns: sequence ID, sense, and anti-sense strands. 
+2. Press the **Convert** button and receive the modified sequence in HELM or custom synthesizer format.
+
+### Supported modifications
```

Comment:
This dataframe is specific to the **Pattern** batch conversion and is not used elsewhere. So, I assume the section header is correct.

Fixed description to avoid misunderstanding.